### PR TITLE
Update collect-diagnostics.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3852

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/remote-actions/collect-diagnostics.md [hunk 1]
```diff
@@ -79,9 +79,9 @@ To use the *Collect diagnostics* action:
 To view the diagnostics collected after an Autopilot failure:
 
 1. Sign in to the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431)
-2. Navigate to **Devices** > **Monitor** > **Autopilot deployments (preview)**.
-3. In the middle pane, select a device.
-4. On the right hand **Properties** pane, under **Device Diagnostics**, select **Download**.
+2. Navigate to **Devices** > **Windows Devices** 
+3. select a device.
+4. Click on **Diagnostics**, select **Download**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/collect-diagnostics.md:84]:
```suggestion
2. Navigate to **Devices** > **Windows Devices**.
3. Select a device.
4. Select **Diagnostics** > **Download**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Brenduns Please review and signoff this PR. Thanks

**Comment by Brenduns** [comment]
#sign-off    @yogkumgit

**Pull request merged**